tests/python/integration/test_int_polling.py

```diff
@@ -0,0 +1,37 @@
+# #import sys
```

Comment:
Why is this file commented out? Does it need to be deleted?

Reply:
Nope. I have created dummy files for integration testing which is used by @cgmannes to develop his tests.